server/controllers/bookController.js

```diff
@@ -51,9 +54,51 @@ class BookController {
         log(error.message);
         Utils.errorStatus(res, 500, error.message);
       }
-    } else {
-      return Utils.successStatus(res, 200, 'Books fetched successfully', await models.Books.findAll());
+    } 
+    if (req.query) {
```

Comment:
I totally do not agree with your logic here.
So from what I can see, if the query has page, it will run the first part, And if it has query, it will run the second part.

WHat happens when it has both page, search and author query?

Reply:
I agree with you, I did not look at it from that angle before now.

I will change the logic to run any query supplied at once.

Reply:
We could have a sync to get more clarity if you need to

Reply:
Sure, I will reach out when I need clarity and please look out for my help request if I get stuck as well.
Thank you